src/NServiceBus.TransportTests/NServiceBusTransportTest.cs

```diff
@@ -208,6 +206,8 @@ static string GetTestName()
         static string MsmqDescriptorKey = "MsmqTransport";
         static string TestIdHeaderName = "TransportTest.TestId";
 
+        static Lazy<List<Type>> transportDefinitions = new Lazy<List<Type>>(() => TypeScanner.GetAllTypesAssignableTo<TransportDefinition>().ToList());
```

Comment:
Ah, I copied that from the acceptance tests as well. You're right, probably doesn't need to be lazy.